documentation/src/main/wiki/BestPractices.md

```diff
@@ -0,0 +1,218 @@
+## Best practices and tips for writing AET test scripts
+
+#### Learn the basics of Selenium WebDriver and understand how to use this knowledge when working with AET
+AET interacts with web pages through WebDriver so it's good to know how to use WebDriver. 
+Knowing about it's limitations and what it requires from test scripts to work properly can help a lot when 
+writing test scripts for AET. It's especially important to understand that in order to get stable tests 
+AET scripts should include smartly located wait operations, just like WebDriver test scripts do.
+
+#### Read the documentation
+AET currently has a lot of useful commands described in the GitHub docs. 
+It's good to know them if you don't want to reinvent the wheel.
+
+#### If AET doesn't have the feature you need, you can suggest it or write it yourself
+You can either write a module and include it in your local AET instance via OSGi or you can contribute 
+to the open source community and create a pull request to the AET code repository.
+
+####  The order of commands matters
+Each command in a test script can influence the state of the browser or the page under test, 
+so it's important to experiment with different variations to get the results you're looking for.
+
+##### Example #1 
+A test can provide different results depending on whether first the browser 
+window is opened or it's target resolution is defined.
+
+###### Script A
+```xml
+<test>
```

Comment:
Maybe it's worth to mention, that this tip concerns `collect` phase only?

Reply:
I'm moving modifier-specific tips to the modifier pages, so this info won't be needed then.